src/services/resources/logic.ts

```diff
@@ -26,6 +27,20 @@ export const getLoggedInLogic = async (userId: string) : Promise<ILogicResponse
     return response;
 };
 
+export const logoutLogic = async () : Promise<ILogicResponse | IErrorResponse> => {
+    const clearCookies: IClearCookie[] = [
```

Comment:
I would overload the existing cookies type to indicate that cookies should be cleared